stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/channels/ChannelsScreen.kt

```diff
@@ -138,13 +146,25 @@ public fun ChannelsScreen(
             }
         }
 
-        val selectedChannel = selectedChannel
-        if (selectedChannel != null) {
+        val selectedChannel = selectedChannel ?: Channel()
```

Comment:
A small note is that due to an empty instance of `Channel` being on the right hand side of the Elvis operator `DefaultChannelInfoHeaderContent` inside `ChannelInfo` will simply instantly pop out before the `ChannelInfo` itself fades and slides out.

This is negligible and there seems to be a lot of redundant code necessary for it to not pop out so I don't think we should insist on addressing it.